### DISCUSSION TITLE
Comparing Overall Chromosomal Accessibility Among Samples

### DISCUSSION THREAD

**Discussion by revolvefire** [stuart-lab/signac#1725] [Q&A]
Thanks for the wonderful tool.

I have multiple samples taken at different time points. After integrating and normalizing the samples together, in addition to analyzing the specific chromatin accessibility profiles of a gene or a region, I would love to know whether the general accessibility level (so at bulk level per sample) changes among the different time points.

What I want is a ChIPseeker coverage plot-like display and I eventually want to compare each plot per sample with Signac normalized peak levels. So that I can visualize/compare each sample's general chromatin accessibility side by side and know that they are directly comparable. 

<img width="948" alt="image" src="https://github.com/stuart-lab/signac/assets/110012834/b3a81489-3b5b-4a06-b0d7-e1a6bdf03e11">

Using the CoveragePlot function in Signac, I can visualize an entire chromosome (e.g., chromosome 1) and compare between samples using the Idents function. However, I have to do this chromosome by chromosome. It takes quite a long time, and the overall window parameter and peak sizes make it difficult to create peak plots that really stand out visibly. 

CoveragePlot( object = obj, region = "chr1-2000-180000000", annotation=FALSE, peaks = FALSE)

<img width="1142" alt="image" src="https://github.com/stuart-lab/signac/assets/110012834/d6a0c15e-30e8-425f-b601-60498e529fa5">

Modulating ymax and other parameters is proving to be somewhat helpful but I was wondering whether you could perhaps give me any suggestions? 

1. Can I somehow make the plot so that all chromosomes are displayed at once. I tried using the granges object with combined peaks that I used to integrate the object, but seems like coverageplot does not take the list of regions.

2. Would there be a better way to visualize what i want (even chromosome by chromosome level) with the current coverage plot function so that the peaks become more visible even at this large scale of genomic range?

3. Would there be any other way of visualizing what I want from the current Signac dataframe that you could perhaps think of? 

Thanks!

**Comment by timoast** [comment]
I don't think there is a great way to do this using Signac, the CoveragePlot function was designed to plot a small region of the genome rather than coverage across the whole genome. You might be better off creating a bigwig file for your data and using other tools

**Reply by revolvefire** [reply]
Got it! Thanks for the prompt response!
